aiplatform/src/test/java/aiplatform/FeatureValuesSamplesTest.java

```diff
@@ -172,7 +171,7 @@ public void tearDown()
     System.setOut(originalPrintStream);
   }
 
-  @Ignore 
+  @Ignore
```

Comment:
Why is this test being ignored?

Reply:
I think there is a flaky setup. We haven't had chance to revisit it.

Reply:
All sample code should have a testcase included. Since, the test doesn't involve the sample in this PR, feel free to merge it. But suggest to create a separate issue and follow up on the testcase.